apps/cowswap-frontend/src/common/updaters/orders/UnfillableOrdersUpdater.ts

```diff
@@ -225,6 +223,9 @@ async function _getOrderPrice(
 
   const amount = getRemainderAmount(order.kind, order)
 
+  // Don't quote if there's nothing left to match in this order
+  if (amount === '0') return null
```

Comment:
Bonus: fix for [this issue](https://cowservices.slack.com/archives/C0361CDD1FZ/p1704472908642609) I observed while testing.

The order would match but still be in the pending list.
This prevent from quoting it and getting unnecessary 400s.

Reply:
Cool, thanks